docs/Adding-Software-to-Homebrew.md

```diff
@@ -156,7 +183,16 @@ cask "my-new-cask" do
   desc ""
   homepage ""
 
+  livecheck do
+    url ""
+    strategy ""
+  end
+
+  depends_on macos: ""
+
   app ""
+
+  zap trash: ""
 end
```

Comment:
This section is getting removed from the docs in my PR based on feedback, so not sure if it's worth adding this here:

- https://github.com/Homebrew/brew/pull/16327

I think a more useful place would be to update the template `brew create` uses, as that should really be the canonical source of generating this.